### PR TITLE
Cutscene Editor and Schema

### PR DESCRIPTION
**Pull request by AndrewJakubowicz**
- Cutscene Editor
 - Refactor Cinematic api endpoints
 - Update the list view of the cinematic editor
 - Use id's or slugs to get cutscenes or cinematics

### CODE REVIEW

File: ozaria/site/components/cutscene/PageCutsceneEditor/index.vue [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+      alert('You must be logged in as an admin to use this page.')
+      return application.router.navigate('/editor', { trigger: true })
+    }
+    this.loadTreema(this.slugOrId)
+  },
+  methods: {
+    async loadTreema(slugOrId) {
+      try {
+        this.cutscene = new Cutscene(await getCutscene(slugOrId))
+        debugger
```
Comment (shubhi1092) [inline ozaria/site/components/cutscene/PageCutsceneEditor/index.vue]: Left by mistake?

File: ozaria/site/components/cutscene/PageCutsceneEditorList/index.vue [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+      alert('You must be logged in as an admin to use this page.')
+      return application.router.navigate('/editor', { trigger: true })
+    }
+    this.fetchList()
+  },
+  methods: {
+    async fetchList () {
+      try {
+        this.cutscenes = await getAllCutscenes()
+        console.log(this.cutscenes)
```
Comment (shubhi1092) [inline ozaria/site/components/cutscene/PageCutsceneEditorList/index.vue]: Left by mistake?

File: app/core/vueRouter.js [hunk 3]
```diff
@@ -29,6 +31,15 @@ export default function getVueRouter () {
           component: PageCinematicEditor,
           props: true
         },
+        {
+          path: '/editor/cutscene',
+          component: PageCutsceneEditorList
+        },
+        {
+          path: '/editor/cutscene/:slugOrId',
```
Comment (shubhi1092) [inline app/core/vueRouter.js]: Nitpick: We can use a single path for `/editor/cutscene` and then subpath for `/:slugOrId` similar to school admin routes.

File: ozaria/site/components/cutscene/PageCutsceneEditor/index.vue (hunk 1 above)
Comment (AndrewJakubowicz) [reply ozaria/site/components/cutscene/PageCutsceneEditor/index.vue]: Thank you!

File: app/core/vueRouter.js (hunk 3 above)
Comment (AndrewJakubowicz) [reply app/core/vueRouter.js]: That's true. Does this require an additional refactor to use `<router-view>`?
I might punt this refactor.

**Comment by AndrewJakubowicz** [comment]
@shubhi1092 Thank you so much for the feedback!

**Pull request merged**